src/templates/nl/base.html

```diff
@@ -88,6 +88,18 @@
 {% block author %}Auteur{% endblock %}
 {% block authors %}Auteurs{% endblock %}
 
+{% block reactions %}Reacties{% endblock %}
+{% block source %}Bron bekijken{% endblock %}
+
+{% block like %}like{% endblock %}
+{% block repost %}repost{% endblock %}
+{% block reply %}antwoord{% endblock %}
+{% block mention %}vermeld{% endblock %}
```

Comment:
```suggestion
{% block mention %}vermelding{% endblock %}
```

Reply:
Missed this. Will open another PR. Happy with the rest of the Dutch? I just used Google Translate to be honest but figured an attempt was better than no attempt at all!

Reply:
Dutch speakers will probably understand 'like' and 'repost' though it's not ideal. Also have some changes for the French translation.